extension-manifest-v3/src/background/stats.js

```diff
@@ -16,13 +17,20 @@ import { getOffscreenImageData } from '@ghostery/ui/wheel';
 import { order } from '@ghostery/ui/categories';
 
 import Request from './utils/request.js';
-import { getMetadata } from './utils/trackerdb.js';
+import * as trackerDb from './utils/trackerdb.js';
 import Options, { observe } from '/store/options.js';
 import AutoSyncingMap from './utils/map.js';
 
-// If you bump this number, the extension will start with a
-// clean state. Normally, this should not be needed.
 const tabStats = new AutoSyncingMap({ storageKey: 'tabStats:v1' });
+const dailyStats = new AutoSyncingMap({
```

Comment:
not sure if this kind of storage is suitable for the data we put here. AutoSyncingMap pushes all its storage to `chrome.storage.local` on every update. That means it gets slower with the amount of data.

Reply:
As I wrote above, the data is limited to less than 2k items with a rather simple structure. As the local storage keeps an object instance to keep data, there is no other way to update the whole object.

However, I doubt that saving 2k items can take a long time (as it is done in "idle" - the action does not block anything) - it can be longer. If we really want that, we can measure it. 

Also, we can extend hard and soft times of sync, so it will happen less often.